js/web/lib/wasm/jsep/webgpu/ops/transpose.ts

```diff
@@ -35,13 +35,18 @@ const permFunctionBody = (perm: number[], rank: number, input: IndicesHelper, ou
   return reverseFunc.join('\n');
 };
 
-export const createTransposeProgramInfo =
-    (inputDataType: number, inputRank: number, permAttr: number[]): ProgramInfo => {
-      const perm = getAdjustedPerm(inputRank, permAttr);
-      const output = outputVariable('output', inputDataType, (permAttr && permAttr.length) || inputRank);
-      const input = inputVariable('a', inputDataType, inputRank);
+export const createTransposeProgramInfo = (inputTensor: TensorView, permAttr: number[]): ProgramInfo => {
```

Comment:
@qjia7  This change requires changes in other kernels using Transpose. Once this change is checked-in other kernels may be fixed. Did sam-b-encode work with this change?

Reply:
Great catching! I only run transpose tests and forgot to run others. Now everything is fixed. Thanks.